### PR TITLE
Acrolinx Batch 2 Edits

### PR DESCRIPTION
**Pull request by daphnemamsft**

### CODE REVIEW

File: docs-ref-conceptual/azure-cli-vm-tutorial-4.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ ms.custom: devx-track-azurecli, seo-azure-cli
 keywords: azure cli show vm information, queries in azure cli
 ---
 
-# 4 - Get VM information with queries
```
Comment (dbradish-microsoft) [inline docs-ref-conceptual/azure-cli-vm-tutorial-4.md:15]: @daphnemamsft , even if Acrolinx doesn't like it, the `#4` shouldn't be replaced with `four`.  In this case, `4` is being used as a list number, not in a sentence.

**Review by dbradish-microsoft — CHANGES_REQUESTED** [review]

File: docs-ref-conceptual/authenticate-azure-cli.md [hunk 2]
```diff
@@ -15,14 +15,12 @@ keywords: az login, authentication types , authentication methods, azure, cli lo
...
+Locally, you can sign in interactively through your browser with the [az login](/cli/azure/reference-index#az-login) command. When you write scripts, the recommended approach is
 to use service principals. By granting just the appropriate permissions needed to a service principal, you can keep your automation secure.
 
-None of your login information is stored by Azure CLI. Instead, an [authentication refresh token](/azure/active-directory/develop/v1-id-and-access-tokens#refresh-tokens)
-is generated by Azure and stored. As of August 2018 this token is revoked after 90 days of inactivity, but this value can be changed by Microsoft or your tenant administrator. Once the token is revoked
-you get a message from the CLI saying you need to login again.
+Azure CLI doesn't store any of your login information. Instead, Azure CLI generates and stores a [authentication refresh token](/azure/active-directory/develop/v1-id-and-access-tokens#refresh-tokens). As of August 2018 this token is revoked after 90 days of inactivity. Contact your Microsoft or tenant administrator to change this value. Once the token is revoked, you get a message from the CLI saying you need to log in again.
```
Comment (dbradish-microsoft) [inline docs-ref-conceptual/authenticate-azure-cli.md:21]: @daphnemamsft , Acrolinx is confused.  `an authentication` is correct.  Acrolinx is not overlooking the `[`.

File: docs-ref-conceptual/authenticate-azure-cli.md [hunk 3]
```diff
@@ -133,7 +131,7 @@ _ Support for Windows Hello, conditional access policies, and FIDO keys.
 _ Streamlined single sign-on.
 _ Bug fixes and enhancements shipped with Windows.
 
-Signing in with WAM is a preview, opt-in feature. Once enabled, it will replace the previous browser-based user interface. 
+Signing in with WAM is a preview, opt-in feature. Once enabled, replaces the previous browser-based user interface.
```
Comment (dbradish-microsoft) [inline docs-ref-conceptual/authenticate-azure-cli.md:134]: consider `Once enabled, the previous browser-based user interface is replaced.`  `replaces` doesn't make sense by itself.

File: docs-ref-conceptual/authenticate-azure-cli.md [hunk 4]
```diff
@@ -142,8 +140,8 @@ az login
 ```
 
 At the current stage of development, there are a few known limitations to WAM:
-_ WAM is available on Windows 10   and later, and on Windows Server 2019 and later. On Mac , Linux, and earlier versions of Windows, we automatically fall back to a browser.  
-_ Microsoft Accounts (for example @outlook.com or @live.com) are not supported for the time being  . We are working with the Microsoft  Identity team to bring the support in the near future.
+_ WAM is available on Windows 10   and later, and on Windows Server 2019 and later. On Mac, Linux, and earlier versions of Windows, we automatically fall back to a browser.  
+_ Microsoft Accounts (for example @outlook.com or @live.com) aren't supported for the time being. We're working with the Microsoft  Identity team to bring the support soon.
```
Comment (dbradish-microsoft) [inline docs-ref-conceptual/authenticate-azure-cli.md:144]: replace `in the near future` with `later`.

remove extra space between `Microsoft` and `Identity`

File: docs-ref-conceptual/azure-cli-vm-tutorial-4.md [hunk 5]
```diff
@@ -21,11 +21,11 @@ Now that a VM has been created, detailed information about it can be retrieved.
 az vm show --name $vmName --resource-group $resourceGroup
 ```
 
-You'll see a lot of information, which can be difficult to parse visually. The returned JSON contains information on authentication, network interface storage,
+You see numerous information, which can be difficult to parse visually. The returned JSON contains information on authentication, network interface storage,
```
Comment (dbradish-microsoft) [inline docs-ref-conceptual/azure-cli-vm-tutorial-4.md:24]: "You see numerous information, which can be difficult to parse visually." needs a complete rewrite.  What are we trying to convey here?  A:  "Virtual Machines have many properties which can be difficult to parse visually."

**Review by dbradish-microsoft — CHANGES_REQUESTED** [review]

File: docs-ref-conceptual/azure-cli-vm-tutorial-4.md (hunk 1 above)
Comment (daphnemamsft) [reply docs-ref-conceptual/azure-cli-vm-tutorial-4.md]: I have revised the number heading

File: docs-ref-conceptual/authenticate-azure-cli.md (hunk 2 above)
Comment (daphnemamsft) [reply docs-ref-conceptual/authenticate-azure-cli.md:21]: This has been revised

File: docs-ref-conceptual/authenticate-azure-cli.md (hunk 4 above)
Comment (daphnemamsft) [reply docs-ref-conceptual/authenticate-azure-cli.md:144]: Revised

**Pull request merged**